src/components/implementation/no_interface/cbboot/quarantine.c

```diff
@@ -144,60 +148,64 @@ quarantine_fork(spdid_t spdid, spdid_t source)
 
 	printd("quarantine_fork(%d, %d)\n", spdid, source);
 
-	if (!(old_sect_cbufs = cos_vect_lookup(&spd_sect_cbufs, source)))
+	if (!(old_sect_cbufs = cos_vect_lookup(&spd_sect_cbufs, source)))					// no idea what either of these is
```

Comment:
I think that you do know now, so this comment is out-dated.  This should point out why it isn't necessarily a good policy to spray your questions throughout the code.  A research log is better for this.